src/visor/rpc.go

```diff
@@ -33,21 +34,23 @@ func (rpc RPC) GetBlockchainMetadata(v *Visor) *BlockchainMetadata {
 }
 
 // GetUnspent gets unspent
-func (rpc RPC) GetUnspent(v *Visor) coin.UnspentPool {
-	return v.Blockchain.GetUnspent().Clone()
+func (rpc RPC) GetUnspent(v *Visor) *blockdb.UnspentPool {
+	return v.Blockchain.Unspent
```

Comment:
i think this intentionally returned a copy of the unspent to avoid certain kinds of errors